lib/calabash/cli/run.rb

```diff
@@ -80,6 +80,17 @@ def run(application_path, cucumber_arguments)
           raise "Invalid platform '#{@platform}'"
         end
 
+        begin
+          require 'cucumber'
+        rescue LoadError => _
+          $stderr.puts "Warning! Could not load cucumber. Make sure it is installed."
+
+          if Object.const_defined?(:Bundler)
+            $stderr.puts "Is cucumber added to your Gemfile?"
+          end
+        rescue Exception => _
```

Comment:
-1 for having a run script.  But that is neither here nor there.

re: catching `Exception` - Why?